modules/ROOT/pages/6.4-release-notes.adoc

```diff
@@ -0,0 +1,1252 @@
+
+> What changes were made to the Enhanced Media Embed plugin? The
+> [changelog][9] contains no entries beyond 2022-09-08 and the only [commits
+> this year][10] have been dependabot bumps or back-end changes.
+
+[8]: <https://notion.so/TinyMCE-6-4-Bill-of-materials-BOM-885a34e3b9884be0b69cd30eba7c618b?d=1c6d570eebbf4a8697d959425fa02808#476b7ddffb464f6c8ffbe9e98db4af84>
+
+[9]: <https://github.com/tinymce/pillow/blob/main/CHANGELOG.md>
+
+[10]: <https://github.com/tinymce/pillow/commits/main>
+////
+
+
+For information on the **Enhanced Media Embed** plugin, see: xref:introduction-to-mediaembed.adoc[Introduction to Enhanced Media Embed].
+
+
+=== Spell Checker Pro 3.2.0
+
+The {productname} 6.4 release includes an accompanying release of the **Spell Checker Pro** premium plugin.
+
+**Spell Checker Pro** 3.2.0 includes the following fixes, additions, and improvements.
+
+////
+To quote my [comment, left on the TinyMCE 6.4 BoM][11] in Notion
+
+> What changes were made to the Enhanced Media Embed plugin? The
+> [changelog][12] contains no entries beyond 2022-09-08 and the only [commits
+> this year][13] have been dependabot bumps or back-end changes.
+
+[11]: <https://notion.so/TinyMCE-6-4-Bill-of-materials-BOM-885a34e3b9884be0b69cd30eba7c618b?d=40e8ab59323f4319bfbb83898a3fe18c#0c32239916564963850ca08c2ab4f112>
+
+[12]: <https://github.com/tinymce/powerspell/blob/main/CHANGELOG.md>
+
+[13]: <https://github.com/tinymce/powerspell/commits/main>
+////
+
+
+For information on the **Spell Checker Pro** plugin, see: xref:introduction-to-tiny-spellchecker.adoc[INtroduction to Spell Checker Pro].
+
+
+=== Advanced Code Editor 3.2.0
+// bforte this is related to https://ephocks.atlassian.net/browse/DOC-1907
+////
+To quote my [comment, left on the TinyMCE 6.4 BoM][11] in Notion
+
+> What changes were made to the Advanced Code Premium plugin? The
+> [changelog][12] contains the below:
+> New button to copy the current dialog content. #TINY-9525
+> New buttons to increase and decrease the font size in the dialog. #TINY-9526
+> New button to toggle dark mode. #TINY-9527
+> New button to toggle fullscreen. #TINY-9529
+
+[14]: <https://github.com/tinymce/advcode/blob/main/CHANGELOG.md>
+
+[15]: <https://github.com/tinymce/advcode/commits/main>
+////
+
+[[accompanying-premium-skins-and-icon-packs-changes]]
+== Accompanying Premium Skins and Icon Packs changes
+
+The {productname} 6.4 release includes an accompanying release of the **Premium Skins and Icon Packs**.
+
+=== Premium Skins and Icon Packs
+
+The **Premium Skins and Icon Packs** release includes the following updates:
+
+The **Premium Skins and Icon Packs** were rebuilt to pull in the changes also incorporated into the default {productname} 6.4 skin, Oxide.
+
+For information on using premium skins and icon packs, see: xref:premium-skins-and-icons.adoc[Premium Skins and Icon Packs].
+
+
+[[improvement]]
+== Improvement
+
+{productname} 6.4 also includes the following improvement:
```

Comment:
```suggestion
{productname} 6.4 also includes the following improvements:
```

Reply:
Arguably the heading should be plural as well, since the other headings (additions, changes) are.